src/schema/schema.graphql

```diff
@@ -2002,6 +2019,12 @@ type Query {
 		folders: String,
 		includeGal: Boolean
 	): AutoCompleteResponse
+	autoCompleteGAL(
+		limit: Int
+		name: String
+		type: String
+		needExp: Int
```

Comment:
Boolean

Reply:
In api docs, this attribute has value 0 | 1. Should it be changed to Boolean?

Reply:
Yes, api will never specify boolean as it is soap api, it will always use 0/1

Reply:
Boolean used instead of Int and pushed